etc/simple-scan.profile

```diff
@@ -20,9 +20,11 @@ nonewprivs
 noroot
 nosound
 notv
-protocol unix,inet,inet6
+novideo
+protocol unix,inet,inet6,netlink
```

Comment:
It won't find the scanner without netlink. I guess that will be the same for all SANE frontends, irrespective of platform.